source/initial_temperature/lithosphere_mask.cc

```diff
@@ -44,6 +44,7 @@ namespace aspect
         {
           const std::string filename = data_directory+LAB_file_name;
 
+          if (Utilities::MPI::this_mpi_process(this->get_mpi_communicator()) == 0)
```

Comment:
Instead of adding this line, change the next one to:
```
this->get_pcout() << ...
```
`pcout` is made to do exactly what you do with the if above (it stands for parallel cout).